src/Util/Player/LivePlayerDataBuilder.php

```diff
@@ -26,7 +26,11 @@ public function buildStreamingData(): array
             OpencastAPI::RETURN_ARRAY
         );
 
-        $media_package = $episode_data['search-results']['result']['mediapackage'];
+        if(array_key_exists('search-results', $episode_data)) {
```

Comment:
@okaufman see my general comment. if we do this, it would be great if you could mention in a comment in the code why and when (for which opencast version) we did this, so that we remember in many, many years why it is the way it is :-)